components/service/sync-logins/src/main/java/mozilla/components/service/sync/logins/AsyncLoginsStorage.kt

```diff
@@ -386,13 +384,6 @@ open class AsyncLoginsStorageAdapter<T : LoginsStorage>(private val wrapped: T)
         fun forDatabase(dbPath: String): AsyncLoginsStorageAdapter<DatabaseLoginsStorage> {
             return AsyncLoginsStorageAdapter(DatabaseLoginsStorage(dbPath))
         }
-
-        /**
-         * Creates an [AsyncLoginsStorage] that is backed by a [MemoryLoginsStorage].
-         */
-        fun inMemory(items: List<ServerPassword>): AsyncLoginsStorageAdapter<MemoryLoginsStorage> {
```

Comment:
The a-s `MemoryLoginsStorage` class is deprecated. I couldn't find any consumers of `inMemory` so I just removed it, but I could change this to be deprecated as well if that's preferred.

Reply:
I think it's fine to remove it. We haven't been very strict with our "deprecate major api pieces before outright removal" policy...
Thanks for mentioning an alternative in the changelog.